testing/src/test/java/io/stargate/it/http/DocumentApiTest.java

```diff
@@ -0,0 +1,1092 @@
+package io.stargate.it.http;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import io.stargate.auth.model.AuthTokenResponse;
+import io.stargate.db.ClientState;
+import io.stargate.db.Persistence;
+import io.stargate.db.QueryState;
+import io.stargate.db.datastore.DataStore;
+import io.stargate.db.datastore.ResultSet;
+import io.stargate.it.BaseOsgiIntegrationTest;
+import io.stargate.it.http.models.Credentials;
+import io.stargate.it.storage.ClusterConnectionInfo;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import net.jcip.annotations.NotThreadSafe;
+import okhttp3.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.osgi.framework.InvalidSyntaxException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@NotThreadSafe
+@Disabled
```

Comment:
Disabled?

Reply:
This is the V1 test, which was previously @Ignore d when we were using junit4, I believe the equivalent in junit5 is @Disabled